doc/cookbook/chatgpt-bot.md

```diff
@@ -0,0 +1,190 @@
+# Build A Telegram ChatGPT Bot With Easegress
+
+Before reading this article, please read the [Multiple API Orchestration](./translation-bot.md) article to ensure understanding of Easegress API orchestration and Telegram robot.
+
+In this article, we will introduce how to build a Telegram ChatGPT robot with Easegress. This robot can generate a response based on the user's input.
```

Comment:
Both `bot` and `robot` are used in this article, per telegram, we should only use `bot`.